meeting-notes/2018/2018-08-13--golang-core-dev-team-weekly.md

```diff
@@ -0,0 +1,184 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 August 13, 2018
+
+- **Lead:** David Dias
+- **Notetaker:** @keks
+- **Attendees:**
+  - @stebalien
+  - @diasdavid
+  - @bigs
+  - @Magik6k
+  - @keks
+  - @djdv
+  - @hsanjuan
+  - @johnnycrunch
+  
+- **Recording:** Recording is currently private to the Go Core Dev Team.
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@stebalien
+ - Done:
+   - Reviewed a bunch of PRs
+   - Responded to/triaged a bunch of issues
+   - Fixed a few bugs
+   - Notable: update gogo protobuf
+ - Blocked:
+   - https://github.com/multiformats/multiaddr/pull/68 <- @daviddias P3
+   - https://github.com/multiformats/multiaddr/pull/68#issuecomment-401971665
+ - Next:
+   - More of the same
+     - CIDv1
+   - Finish the protocol negotiation spec
```

Comment:
@Stebalien remind me, is this related with Multistream?

Reply:
Yes. It's the "next" version of protocol negotiation.

Reply:
Interesting. I didn't know that was urgent/`P0`

Reply:
The sooner we fix it, the better. Every other protocol we have relies on it and we currently (in go at least) try rather hard to reuse streams as creating new ones is expensive (in terms of bandwidth).

For context, inefficient stream negotiation means we need things like https://github.com/libp2p/go-libp2p-kad-dht/pull/92 instead of https://github.com/libp2p/go-libp2p-kad-dht/pull/167.